src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py

```diff
@@ -908,22 +949,31 @@ def utask_main(uworker_input):
   if is_lsan_enabled:
     leak_blacklist.copy_global_to_local_blacklist()
 
+  uworker_output = None
   try:
     result = do_corpus_pruning(context, revision)
-    _record_cross_pollination_stats(result.cross_pollination_stats)
-    _save_coverage_information(context, result)
+    uworker_output = uworker_msg_pb2.Output(
+        corpus_pruning_task_output=uworker_msg_pb2.CorpusPruningTaskOutput(
+            coverage_info=_extract_coverage_information(context, result)))
+    _fill_cross_pollination_stats(result.cross_pollination_stats,
+                                  uworker_output)
```

Comment:
nit (follow-up): and then you could have something consistent here?
```suggestion
    uworker_output = uworker_msg_pb2.Output(
        corpus_pruning_task_output=uworker_msg_pb2.CorpusPruningTaskOutput(
            coverage_info=_extract_coverage_information(context, result), 
            cross_pollination_stats=_extract_cross_pollination_stats(result.cross_pollination_stats)))
```

Reply:
Not relevant as per Ali's comment above.